### ISSUE TITLE
Issue with gog games

### ISSUE THREAD

**Issue by Biniou180** [lutris/lutris#2173]
Hi, 

I have some issues when trying to install GOG games with Lutris: The installers stay stuck indefinitely with the message `Waiting on children \ All children gone \ Exit with returncode 0`. So far I've tried with [Divinity Original Sin 2](https://lutris.net/games/divinity-original-sin-2/) and [Gris](https://lutris.net/games/gris/).

I'm on a fresh installed fedora 30 with Lutris 0.5.2.1. I've seen on previous issues ( #2089, #1949 ) that it could be related to innoextract, but i don't know how to confirm that or change the innoextract version (sorry, i'm quite new to lutris)

Could you please help to solve this issue?
Thanks a lot :)

Here are the detailed logs: 
**Json file issued by --submit-issue command**: https://framabin.org/p/?1f7d5bfb325694ee#LLKwL2Jhw98KnOIYyu2UcCZXmKowazeg6YLELFrc++0=

**Divinity 2 output**: https://framabin.org/p/?b2cc7d164a2b6027#5OKl7hkIGs9eXEmgmL9Yx8ytYvng1/C8z9baUECqHDY=

**Gris output**:
```
Running /home/biniou/.local/share/lutris/runtime/winetricks/winetricks --unattended mf
------------------------------------------------------
You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
Using winetricks 20190310-next - sha256sum: ba32734c7e69ae53929d92184fadcea66836cd5c9ba43694e153b53522d74669 with wine-4.9 (Staging) and WINEARCH=win64
Executing w_do_call mf
Executing load_mf 
Executing cabextract -q -d /home/biniou/Jeux/Lutris/gris/dosdevices/c:/windows/temp/_mf -L -F x86_microsoft-windows-mediafoundation_31bf3856ad364e35_6.1.7601.17514_none_9e6699276b03c38e/mf.dll /home/biniou/.cache/winetricks/win7sp1/windows6.1-KB976932-X86.exe
Executing cp /home/biniou/Jeux/Lutris/gris/dosdevices/c:/windows/temp/_mf/x86_microsoft-windows-mediafoundation_31bf3856ad364e35_6.1.7601.17514_none_9e6699276b03c38e/mf.dll /home/biniou/Jeux/Lutris/gris/dosdevices/c:/windows/syswow64/mf.dll
Executing cabextract -q -d /home/biniou/Jeux/Lutris/gris/dosdevices/c:/windows/temp/_mf -L -F amd64_microsoft-windows-mediafoundation_31bf3856ad364e35_6.1.7601.17514_none_fa8534ab236134c4/mf.dll /home/biniou/.cache/winetricks/win7sp1/windows6.1-KB976932-X64.exe
Executing cp /home/biniou/Jeux/Lutris/gris/dosdevices/c:/windows/temp/_mf/amd64_microsoft-windows-mediafoundation_31bf3856ad364e35_6.1.7601.17514_none_fa8534ab236134c4/mf.dll /home/biniou/Jeux/Lutris/gris/dosdevices/c:/windows/system32/mf.dll
Using native,builtin override for following DLLs: mf
Executing /home/biniou/.local/share/lutris/runners/wine/ge-protonified-4.9-x86_64/bin/wine regedit /S C:\windows\Temp\_mf\override-dll.reg
000b:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
000b:fixme:winediag:start_process Wine Staging 4.9 is a testing version containing experimental patches.
000b:fixme:winediag:start_process Please mention your exact version when filing bug reports on winehq.org.
000f:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
0012:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
0019:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
001f:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
0026:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
Executing /home/biniou/.local/share/lutris/runners/wine/ge-protonified-4.9-x86_64/bin/wine64 regedit /S C:\windows\Temp\_mf\override-dll.reg
002b:fixme:thread:create_user_shared_data_thread Creating user shared data update thread.
Waiting on children
All children gone
Exit with returncode 0
```

**Comment by Biniou180** [comment]
For completion's sake, I've also tried these installers on a up-to-date archlinux on the same PC, same error at the end of the process.
If you need to do tests or have more details, feel free to tell me. 
Thank you :)

**Comment by NickZ** [comment]
Confirming this, I am having this issue as well.

**Comment by NickZ** [comment]
After testing, it appears that this is present in 0.5.2.2, but not on the current master. After bisecting, it looks like commit 84ae51026555763c72188849c11f6564bc555961 solved this issue. After cherrypicking that commit onto the top of the 0.5.2.2 tag, it worked perfectly. I recommend cherrypicking that commit for a minor point release.

**Comment by NickZ** [comment]
@tannisroot can you please take a look at this?

**Comment by Biniou180** [comment]
I confirm as well on my computer: using the master branch on git and installing works without error, the install process goes to the end and the game is playable.
Thanks a lot @NickZ to have identified the problem root!

**Comment by p2004a** [comment]
I've hit the same problem when trying to install two games today with 0.5.2.2. I've trier version on master and I confirm that it indeed doesn't have this problem anymore.

Also looks like #2230 and #2215 are duplicates of this issue.

**Comment by strycore** [comment]
Going to release a patch release containing this fix soon.
